webservices/tasks/utils.py

```diff
@@ -19,3 +22,13 @@ def get_bucket():
 
 def get_object(key):
     return get_bucket().Object(key=key)
+
+def get_s3_key(name):
+    connection = boto.s3.connect_to_region(
```

Comment:
I know you had mentioned this before somewhere but can't remember - `boto3` didn't provide what was needed?  I know both can co-exist, just wanted to make sure I had a clear understanding!